src/Service/BlogService.php

```diff
@@ -5,10 +5,12 @@
 
 class BlogService
 {
-    private static function getClient()
+    private $client;
+
+    public function __construct($blogUrl, $username, $password)
     {
-        $wp = new WordpressClient($_ENV['WP_ENDPOINT'], $_ENV['WP_USER'], $_ENV['WP_PASSWORD']);
-        return $wp;
+        $url = rtrim($blogUrl, "/") . "/xmlrpc.php";
+        $this->client = new WordpressClient($url, $username, $password);
```

Comment:
Ok, sorry I misunderstood that. What about services that don't rely on another class and make the calls themselves? Should they stay as they are?